sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/components/email/html/HtmlNotificationEmail.java

```diff
@@ -71,16 +79,25 @@ public boolean equals(Object o) {
     if (!recipient().equals(that.recipient())) {
       return false;
     }
+    if (!subject().equals(that.subject())) {
+      return false;
+    }
     return body().equals(that.body());
   }
 
   @Override
   public int hashCode() {
     int result = recipient().hashCode();
     result = 31 * result + body().hashCode();
+    result = 31 * result + subject().hashCode();
     return result;
   }
 
+  private String formatSubject(String projectCode) {
+    requireNonNull(projectCode);
+    return String.format("Samples updated in project %s", projectCode);
+  }
+
```

Comment:
why doing that here? I think it would make sense to have this in the SendmailHeader class, where you get the content. You could have a method e.g getSendmailHeaderFor(String project) or so

Reply:
As the subject is email specific, I think this belongs here.